app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -486,6 +489,65 @@ class ProfileChooserFragmentTest {
     }
   }
 
+  @Test
+  @RunOn(TestPlatform.ESPRESSO)
+  fun testProfileChooserFragment_clickProfile_checkOpensHomeActivity() {
+    profileManagementController.addProfile(
+      name = "Admin",
+      pin = "",
+      avatarImagePath = null,
+      allowDownloadAccess = true,
+      colorRgb = -10710042,
+      isAdmin = true
+    )
+    launch<ProfileChooserActivity>(createProfileChooserActivityIntent()).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.profile_recycler_view,
+          position = 0,
+          targetViewId = R.id.profile_chooser_item
+        )
+      ).perform(click())
+      intended(hasComponent(HomeActivity::class.java.name))
+      it.onActivity { activity ->
+        assertThat(
+          activity.intent.extractCurrentUserProfileId().internalId
+        ).isEqualTo(0)
```

Comment:
Same comment about intents, here and below.

Reply:
Done.